src/lib/loaders/loaders_without_authentication/diffusion.js

```diff
@@ -1,11 +1,12 @@
 // @ts-check
 import factories from "../api"
+import config from "config"
 
 export default requestID => {
   const { diffusionLoaderWithoutAuthenticationFactory } = factories(requestID)
   const diffusionLoader = diffusionLoaderWithoutAuthenticationFactory
 
   return {
-    auctionLotLoader: diffusionLoader("lots"),
+    auctionLotLoader: diffusionLoader("lots", {}, { requestThrottleMs: config.DIFFUSION_REQUEST_THROTTLE_MS }),
```

Comment:
Seeing as this env var seems to imply that the timeout should apply to _all_ diffusion requests, maybe this setting should move to `../api/index.js` where `diffusionLoaderWithoutAuthenticationFactory` is defined?

Reply:
Good call 👍

Reply:
Updated.